modules/core/src/test/java/org/apache/ignite/spi/communication/tcp/internal/BlacklistNetwokInterfacesTest.java

```diff
@@ -89,4 +89,18 @@ public void testBlacklistFilter() throws Exception {
             IgniteUtils.resetCachedLocalAddressAndHostNames();
         }
     }
+
```

Comment:
let's add test-case for string like "    " (one or more whitespaces)

Reply:
test added